wat

```diff
@@ -1,6 +1,10 @@
 #!/bin/sh
 log_lines() {
-  local regex='^.*\(starting full system upgrade\|upgraded.*\)$'
+  local installed=''
+  if [ "x$SHOW_INSTALLED" == "x1" ]; then
```

Comment:
I'm strictly a bash fellow, so I'm generally not sure what available in "less featured" shells. If you're happy with it, I'd rather use -eq "1".

As for passing the value in, I'm not sure of a clean way to do it without moving or doing another arg-parse operation? Suggestions?